src/App.tsx

```diff
@@ -1,61 +1,118 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(UserId: number) {
+  const foundUser = usersFromServer.find(user => user.id === UserId);
+
+  return foundUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [userId, setUserId] = useState(0);
+  const [userIdError, setUserIdError] = useState(false);
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+
+  const changeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setTitleError(false);
+    setTitle(value);
+  };
+
+  const changeUserId = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    setUserIdError(false);
+    setUserId(Number(value));
+  };
```

Comment:
According to the checklist better name `Event Handler Functions`  according to [this rules](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)